### PR TITLE
Experiment on a packages structure

### PR DESCRIPTION
**Pull request by nachocodoner**
In order to understand better how Meteor builder works, I attempt to perform an experiment to see how files affect when they live on package, and not the app level. It is closer on how our infrastructure works.

Please, [read the readme](https://github.com/Gywem/build-performance-9642/blob/55d0e9b9781d6a53ce6a7bc06607678ce60569af/private/with-meteor-packages/README.md) to check how the experiment was performed, the analysis and next steps.

### CODE REVIEW

**Comment by Gywem** [comment]
Feels like I can get it compiled now in less seconds, not sure the reason, but I probably going to continue with my tests.

File: private/with-meteor-packages/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+- Importing in the server code a package via the `mainModule` produces to load eagerly all modules involved, adding time to the `ServerStartup` phase.
+
+### Next experiments
+
+Meteor builder seems to not be optimized on use packages. Besides, for the best optimization we should ensure that we load by demand the modules needed and not more than those to alliviate the build process. For these the next experiments would be around:
+
+- Consider to use NPM packages instead of meteor packages here, using lernajs to manage the project.
+- Compile NPM packages with latest Webpack as we may not have support of having NPM ES6 modules to be consumed in Meteor.
+- Include `babel-transform-imports` plugin to translate our index imports to specific ones, provided that [I implemented regex support for it](https://bitbucket.org/amctheatres/babel-transform-imports/pull-requests/8/support-regular-expressions/diff). This would be useful for others apps that are not using all modules but requires to set a convention.
```
Comment (filipenevola) [inline private/with-meteor-packages/README.md:93]: I don't get the performance benefits of it, can you explain to me?

**Review by filipenevola — APPROVED** [review]
Nice job!

**Review by flippyhead — COMMENTED** [review]
This is great.

**Pull request merged**